tracer/test/Datadog.Trace.Tests/Logging/NLogLogProviderTests.cs

```diff
@@ -42,80 +42,32 @@ public NLogLogProviderTests()
             LogManager.Configuration = config;
             SimpleConfigurator.ConfigureForTargetLogging(_target, NLog.LogLevel.Trace);
 
-            _logProvider = new CustomNLogLogProvider();
+            _logProvider = new NoOpNLogLogProvider();
             LogProvider.SetCurrentLogProvider(_logProvider);
             _logger = new LoggerExecutionWrapper(_logProvider.GetLogger("test"));
         }
 
         [Fact]
-        public void LogsInjectionEnabledAddsParentCorrelationIdentifiers()
+        public void LogsInjectionEnabledDoesNotAddCorrelationIdentifiers()
```

Comment:
Can we consolidate the `true`/`false` into this one test case as a `[Theory]`?

Reply:
Almost certainly, and if so, we can prob do the same for the log4net/serilog - pretty sure I copied the changes you made in the log4net PR (or at least, I meant to!)

Reply:
Changed to `[Theory]` in 8e280fbaeb36d290b1788ef949d74e4671193ff6